org-roam.el

```diff
@@ -459,12 +459,46 @@ INFO is an alist containing additional information."
   "Face for org-roam backlinks in backlinks buffer"
   :group 'org-roam-faces)
 
+(defun org-roam--in-buffer-p ()
+  "Return t if in the Org-roam buffer."
+  (string= (buffer-name)
+           org-roam-buffer))
```

Comment:
This isn't a particularly robust way to check if we're in the side-window, but I don't think there was anything allowing this check in the codebase.  It'd probably be a good idea to create a minor-mode for the side-window, and we should keep this in mind when we get to the revamp.

Reply:
You're right.

Addressed in 0d210932fe63c00945f888b3961f6b909c3c3f96.